python/lsst/synpipe/positionStarFakes.py

```diff
@@ -68,7 +68,7 @@ def run(self, exposure, background):
 
             starXY = wcs.skyToPixel(starCoord)
             bboxI = exposure.getBBox(PARENT)
-            bboxI.grow(margin)
+            bboxI.grow(int(margin))
```

Comment:
change the commit message of this from "Misc fixes" to something like "Cast variables to proper types"